src/bindings/c/include/openvino/c/ov_property.h

```diff
@@ -101,6 +101,7 @@ ov_property_key_cache_dir;
  * @brief Read-write property<string> to select the cache mode between optimize_size and optimize_speed.
  * If optimize_size is selected, smaller cache files will be created.
  * And if optimize_speed is selected, loading time will decrease but the cache file size will increase.
+ * This is only supported from GPU.
```

Comment:
I suppose we also modify C++ property definition? https://github.com/openvinotoolkit/openvino/blob/77cab19e565790cb6abffea80ac1b29c2f0dcd75/src/inference/include/openvino/runtime/properties.hpp#L729-L735

Reply:
Thanks for pointing out that!